reana_workflow_controller/app.py

```diff
@@ -24,174 +24,28 @@
 
 from __future__ import absolute_import
 
-import os
-import traceback
+from flask import Flask
 
-from flask import Flask, abort, jsonify, redirect, request
+from .multiorganization import MultiOrganizationSQLAlchemy
 
-from .fsdb import Organization, get_all_workflows
-from .tasks import run_yadage_workflow
+# Initialize DB
+db = MultiOrganizationSQLAlchemy()
 
-app = Flask(__name__)
-app.config.from_object('reana_workflow_controller.config')
-app.secret_key = "super secret key"
+from .models import Tenant  # isort:skip  # noqa
 
-experiment_to_queue = {
-    'alice': 'alice-queue',
-    'atlas': 'atlas-queue',
-    'lhcb': 'lhcb-queue',
-    'cms': 'cms-queue',
-    'recast': 'recast-queue'
-}
 
+def create_app():
```

Comment:
As discussed IRL, let's put factory creation into `factory.py` and let's leave `app.py` to instantiate the application out of it.